docs/code_ref/coordinates/carrington.rst

```diff
@@ -0,0 +1,126 @@
+
+Background information
+----------------------
+
+The IAU definition
+^^^^^^^^^^^^^^^^^^
+
+Since the original definition of Carrington coordinates [#Carrington]_, there have been a number of refinements of the definition, but there have been inconsistencies with how those definitions are interpreted.
+Possible points of contention include the reference epoch, the rotation period, and the handling of observer effects.
+
+The most recent definition was essentially laid down in 2007 by the IAU [#IAU]_:
+
+* The reference epoch is J2000.0 TDB (2000 January 1 12:00 TDB).
+  The difference between Barycentric Dynamical Time (TDB) and Terrestrial Time (TT) is very small, but both are appreciably different from Coordinated Universal Time (UTC).
+* The longitude of the prime meridian (|W0|) at the reference epoch is 84.176 degrees.
+  This longitude is the "true" longitude, without accounting for any effects that would modify the apparent prime meridian for an observer (e.g., light travel time).
+* The sidereal rotation rate is exactly 14.1844 degrees per day.
+  This is close to, but not exactly the same as, a sidereal rotation period of 25.38 days or a mean synodic period of 27.2753 days.
+* The ICRS coordinates of the Sun's north pole of rotation is given as a right ascension of 286.13 degrees and a declination of 63.87 degrees.
+
+The tuning of |W0|
+^^^^^^^^^^^^^^^^^^
+
+Importantly, the IAU parameters (specifically, |W0|) were tuned following an investigation by |AA| [#AA]_.
+The prescription of |AA| included both the correction for light travel time and the correction for stellar aberration due to Earth's motion.
+When using this prescription, |AA| found that a |W0| value of 84.176 degrees minimized the differences between the modern approach and earlier approaches.
+However, for those purposes where one should use sub-observer Carrington longitudes without the stellar-aberration correction, there will be a discrepancy compared to |AA| and to calculations made using earlier approaches.
+
+The approach in SunPy
+---------------------
+
+SunPy determines apparent sub-observer Carrington longitude by including the correction for the difference in light travel time, but **excluding** any correction for stellar aberration due to observer motion.
+The exclusion of a stellar-aberration correction is appropriate for purposes such as image co-alignment, where Carrington longitudes must be consistently associated with features on the Sun's surface.
+
+Comparisons to other sources
+----------------------------
+
+Compared to |AA| and older methods of calculation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+|AA| publishes the apparent sub-Earth Carrington longitude (|L0|), and these values include the stellar-aberration correction.
+Consequently, SunPy values will be consistently ~0.006 degrees (~20 arcseconds) greater than |AA| values, although these discrepancies are not always apparent at the printed precision (0.01 degrees).
```

Comment:
Knowing this is consistent is very good! Nice point about the printed precision!